AsyncDisplayKit/ASMapNode.mm

```diff
@@ -41,11 +40,9 @@ - (instancetype)init
   _liveMap = NO;
   _centerCoordinateOfMap = kCLLocationCoordinate2DInvalid;
   
-  //Default world-scale view
-  _region = MKCoordinateRegionForMapRect(MKMapRectWorld);
-  
   _options = [[MKMapSnapshotOptions alloc] init];
-  _options.region = _region;
+  //Default world-scale view
+  _options.region = MKCoordinateRegionForMapRect(MKMapRectWorld);
```

Comment:
Don't you need to allocate an Options first?  I think this will just fall on "nil" ears :).

The better route is probably to go to the point of use for the _options, and if it is nil there, create one with some default values.  I'd much rather we avoid creating objects in -init in framework classes if they're avoidable via some lazy strategy such as this.